homework/list.js

```diff
@@ -1,79 +1,92 @@
 'use strict';
 
-(function() {
+(function(exports) {
+	var TodoListManager = function() {
+		this._listNoteContent = [];
+		this._wrapper = document.querySelector('#note-list-wrapper');
+	};
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
+	TodoListManager.prototype = {
+		start() {
+			this.fetchList(function(data) {		
+				this.updateList(data)
+				.then((function(){
+					this.drawList(); 
+				}).bind(this)).then((function(){
+					this.preloadFirstNote();
+				}).bind(this)).catch((function(error){
```

Comment:
And please be careful: this is a serious issue which definitely deserve a review- if you're working and set the review for this patch.